### PR TITLE
Contact service modify

### PR DESCRIPTION
**Pull request by tobidub**

### CODE REVIEW

**Comment by AgentEnder** [comment]
@tobidub heads up, your build failed.

File: OtripleS.Web.Api.Tests.Unit/Services/Contacts/ContactServiceTests.Exceptions.Add.cs [hunk 1]
```diff
@@ -5,157 +5,152 @@
 
 using System;
 using System.Threading.Tasks;
```
Comment (viralpandya) [inline OtripleS.Web.Api.Tests.Unit/Services/Contacts/ContactServiceTests.Exceptions.Add.cs:7]: Code format with tab is not usual

File: OtripleS.Web.Api.Tests.Unit/Services/Contacts/ContactServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -11,108 +11,108 @@
 
 namespace OtripleS.Web.Api.Tests.Unit.Services.Contacts
 {
-    public partial class ContactServiceTests
```
Comment (viralpandya) [inline OtripleS.Web.Api.Tests.Unit/Services/Contacts/ContactServiceTests.Exceptions.RetrieveAll.cs:14]: It seems that all the files code format is changed with tabs

**Comment by viralpandya** [comment]
@tobidub : Can you please check code format settings in visual studio? As its not usual and that is changing all the file formats too.

**Comment by tobidub** [comment]
To my knowledge it is tabs over spaces @viralpandya @hassanhabib

**Comment by tobidub** [comment]
> @tobidub heads up, your build failed.

Yeah thats the funny thing, there are a few Test with "ShouldThrowValidationExceptionOnModifyWhenUpdatedDateIsNotRecentAndLogItAsync" which have a 1/150 or 1/90 chance to fail if  randomNumber is choosen as 1... i tried to fixed that bug in #157 but someone changed it back

**Pull request merged**